### PR TITLE
Bump build-linux-rust to 1.77.2:v0.0.4

### PR DESCRIPTION
**Pull request by tomaszklak**
### Problem
Old version was build only for amd64.

### Solution
This version is the first built for both arm64 and amd64.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: .github/workflows/gitlab.yml [hunk 1]
```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v1.2.1
+          triggered-ref: LLT-5248-bump-build-linux-rust
```
Comment (tomaszklak) [inline .github/workflows/gitlab.yml:34]: ❗REMINDER TO EDIT BEFORE MERGE❗

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -15,4 +15,4 @@ libtelio-build-pipeline:
 
   trigger:
     project: $LIBTELIO_BUILD_PROJECT_PATH
-    branch: v1.2.1
+    branch: LLT-5248-bump-build-linux-rust
```
Comment (tomaszklak) [inline .gitlab-ci.yml:18]: ❗REMINDER TO EDIT BEFORE MERGE❗

**Review by Jauler — CHANGES_REQUESTED** [review]
:) triggered-ref` needs to be updated.

**Comment by tomaszklak** [comment]
> :) `triggered-ref` needs to be updated (:

Done ;)

**Review by Jauler — APPROVED** [review]
+2.0

**Pull request merged**